packages/jest-preset-kyt-enzyme/jest-preset.js

```diff
@@ -10,9 +10,9 @@ const jestConfig = {
   setupFilesAfterEnv: [require.resolve('./setup')],
   // if projects spread this value, the import can be lost if it is not absolute
   snapshotSerializers: [require.resolve('enzyme-to-json/serializer')],
-  testMatch: ['**/*.test.js'],
+  testMatch: ['**/*.test.[jt]s?(x)'],
```

Comment:
For reference, here is [Jest's default `testMatch` config](https://jestjs.io/docs/en/configuration.html#testmatch-arraystring):

```json
{
  "testMatch": [ "**/__tests__/**/*.[jt]s?(x)", "**/?(*.)+(spec|test).[jt]s?(x)" ]
}
```